tests/unit/test_replay_manager.py

```diff
@@ -23,7 +23,7 @@ def replay_sim_config_file(ringtest_baseconfig, input_spikes_file):
         "node_set": "RingB",
     }
 
-    with NamedTemporaryFile("w", suffix='.json', delete=False) as config_file:
+    with NamedTemporaryFile("w", suffix='.json', dir=tmp_path, delete=False) as config_file:
         json.dump(ringtest_baseconfig, config_file)
     yield config_file
     os.unlink(config_file.name)
```

Comment:
what happens if there is an exception thrown? should the `os.unlink` be in a `finally`?

Reply:
(this is non-blocking of merge, btw)

Reply:
In case of exception thrown during `json.dump`, it may be useful to look at the broken `config_file` for debugging. Right?
It is coded like that in many places. We could discuss and refactor in another PR.